database/cassandra/index.js

```diff
@@ -0,0 +1,49 @@
+const cassandra = require('cassandra-driver');
+
+var keyspace = process.env.ABOUT_DATABASE.replace(/-/g, '__');
+
+var client = new cassandra.Client({
+  contactPoints: [process.env.CASSANDRA_CONTACTPOINT],
+  localDataCenter: 'datacenter1',
+  keyspace
+});
+
+var queries = {
+  createTable: `CREATE TABLE IF NOT EXISTS courses (
+    course_id bigint,
```

Comment:
I like how you broke the table down into individual columns. I used a single json object, but the advantage of your solution is that you can query by column other than id.

Reply:
To be honest doing it that way would likely have been way faster and easier 😮 I don't really need to search by anything other than course...